documentation/src/main/asciidoc/topics/con_xsite_active_passive.adoc

```diff
@@ -41,3 +43,8 @@ while handling client requests. If either **NYC** or **LON** go offline,
 clients can start sending requests to the online site. You can then bring
 offline sites back online, push state to synchronize data, and switch clients
 as required.
+
+.Backup strategies
+
+With an Active/Active configuration, you should always use an asynchronous backup strategy (`strategy=async`).
```

Comment:
should be a WARNING or a NOTE
Also, sync can be use with active/active if both sites access different data set.